lib/shopify_app/session/session_repository.rb

```diff
@@ -66,8 +71,10 @@ def delete_session(id)
         match = id.match(/^offline_(.*)/)
 
         record = if match
+          ShopifyApp::Utils::Logger.debug("Destroying Session by domain - domain: #{match[1]}")
           Shop.find_by(shopify_domain: match[1])
         else
+          ShopifyApp::Utils::Logger.debug("Destroying Session by user - user_id: #{id.split("_").last}")
```

Comment:
I would avoid doing any manipulation in the log calls, since it could later get out of sync with the implementation. Instead we could extract `id.split("_").last` to a variable and use that in both places.